farm2.def.json

```diff
@@ -0,0 +1,57 @@
+{
+    "version": 2,
+    "name": "Farm 2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Farm3D",
+        "manufacturer": "Farm3D LLC",
+        "file_formats": "text/x-gcode",
+        "platform": "farm2_platform.stl",
+        "platform_texture": "farm2plate.png",
+        "platform_offset": [-7, 0, -4],
+		"has_variants": true,
+		"preferred_variant_name": "PrintHead 0.4 mm",
+		"variants_name": "PrintHead type",
+        "has_materials": true,
+	"preferred_material": "generic_pla",
+        "has_machine_quality": false,
+                "machine_extruder_trains":
+        {
+            "0": "farm2_extruder_1"
+			
+        }
+        
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Farm 2" },
+        "machine_start_gcode" : {
+            "default_value": "G28 ;Home\nG1 Y0 Z3.0 F4000 ;Move the platform\nG92 E0\nG1 F1000 Z0.2\nG1 X190 F1000 E50 Z0.2\nG92 E0"
+        },
+        "machine_end_gcode" : {
+            "default_value": "M104 S0\nM140 S0\nG92 E1\nG1 E-1 F300\nG28\nM84\nM81"
+        },
+        "machine_width": {
+            "default_value": 209
```

Comment:
I couldn't find online what the dimensions of this printer are, but this guy claims 215x215x205mm: https://rozetka.com.ua/104976036/p104976036/comments/

I'll believe you here, but you might want to double-check.

Reply:
hello, the correct printer value is x(203) y(199) z(197)

Reply:
its for farm2

Reply:
for farm2ce - x(209) y(199) z(213)

Reply:
Nice, just wanted to check with you.

Reply:
ok i make changes and add new platform which is used both for 2 printers, but i not sure if they correct open in correct position so i add a screen how they would be
![zone](https://user-images.githubusercontent.com/23043730/121728457-f750d480-caf5-11eb-800e-b5fd54cf4dd6.png)

Reply:
i mean platform offset will now not correct when i change file to 3mf

Reply:
That would depend on the way these files are converted then. At some point I converted all existing STLs to 3MF and that went pretty painlessly for the most part: https://github.com/Ultimaker/Cura/pull/7545